conforming/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2022,2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+FROM alpine:3.18.2
+ARG JAR
+ARG LIB
+
+ARG APP_USER=agent
+ARG APP_UID=10100
+ARG APP_GID=30000
+
+RUN apk update && \
+    apk add openjdk17-jre-headless=17.0.7_p7-r1 --no-cache && \
```

Comment:
Use Eclipse Temurin as base image for Java based applications instead of alpine and adding additional packages to the image. Please refer to [TRG 4.02 - Base images](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-02).

Reply:
switched to eclipse-temurin:17.0.6_10-jre-alpine